doc-content/enterprise-only/cases/case-management-stages-con.adoc

```diff
@@ -9,4 +9,4 @@ There are three ways to complete a stage:
 
 * By completion condition.
 * By terminal end event.
-* By setting the `AdHocCompletionCondition` to `autocomplete`, which will automatically complete the stage when there are no active tasks left in the stage. 
+* By setting the `Completion Condition` to `autocomplete`, which will automatically complete the stage when there are no active tasks left in the stage. 
```

Comment:
the value of the Completion Condition, or something like that.
which automatically completes
These list items should not have periods because none are complete sentences